strings/quotes.txt

```diff
@@ -6,11 +6,13 @@
 "Das ist Daten-Müll!" - Helmut Seidl, 2019
 "Das können sie schon machen, wird dann aber teuer!" - Helmut Seidl, 2019
 "So wenn man das lesen könnte, wärs vielleicht korrekt" - Helmut Seidl, 2019
+"Das Objekt wird sozusagen von niemandem geliebt. Und dann kommt der Garbage Collector!" - Helmut Seidl, 2019
 "PGdP ist äußerst exzellent!" - PGdP, 2019
 "Nope" - Johannes Stöhr, 2019
-"*böses Lachen*" - Johannes Stöhr, Woche 07 H01
+"*böses Lachen*" - Johannes Stöhr, über W07H01, 2019
```

Comment:
While we're at it, could you escape the asterisks in this quote (by adding a backslash in front of them)?

Reply:
@timschumi yup, done.